lets-auth.fifthtry.site/actions/dummy/create-account.ftd

```diff
@@ -29,3 +29,29 @@ show_alert(
     accept_terms,
     next
 )
+
+;; Note: unused in this package.
+;; This is provided to support create account flows where the user is pre verified and they just want to give basic info and set password.
+;; The `code` should be used for verifying genuine requests
+-- void create-account-with-code(name, email, username, password, password2, accept_terms, next, code):
+ftd.string-field $name:
+ftd.string-field $email:
+ftd.string-field $username:
+ftd.string-field $password:
+ftd.string-field $password2:
+ftd.boolean-field $accept_terms:
+ftd.string-field $next:
+string code:
+js: $assets.files.actions.dummy.alert.js
+
+show_alert(
+    "/-/auth/create-account/",
```

Comment:
We still have to hard code this? Can we not make this a variable and populate it with `ftd.app-url()` function?

Reply:
The other one in this PR have to be changed too.